src/lib/RequestTypes.ts

```diff
@@ -259,6 +261,12 @@ export interface ParsedMigrateRequest {
     kind: RequestType.MIGRATE;
 }
 
+export interface ExportResult {
+    success: true;
+    hasFile: boolean;
+    hasWords: boolean;
```

Comment:
I proposed to use `hasFile & hasWords` in the Keyguard, too. Let me know what you think about that.

Reply:
`hasFile` and `hasWords` just doesn't really describe what these flags are actually for.
Every account hast words. Thus, `wordsBackedUp` would be more correct.

Reply:
Hm, you are right. With `hasFile`, I meant _the user has the file_, but since it's stored with the account, you interpret it as _the account has a file_, which is wrong, obviously.

So yeah, the variable name should be changed.

Reply:
How about `isWordsExported` and `isFileExported`?

Reply:
Also ok for me, but should be `areWordsExported` then.
Maybe just `wordsExported` and `fileExported`

Reply:
The `is` of my suggestion is a convention to signal `booleans`. I did not want to use only `wordsExported`, because that can be interpreted again as containing the actual words (or the actual file), instead of a bool flag.